Formula/fn.rb

```diff
@@ -0,0 +1,35 @@
+class Fn < Formula
+  desc "CLI tool for the fn project."
+  homepage "https://fnproject.github.io"
+  url "https://github.com/fnproject/cli/archive/0.3.72.tar.gz"
+  sha256 "42f19a50cfb649fa2677ae1e1bb09847e9eb533a75658446e20d84afb086c71b"
+
+  bottle do
+    cellar :any_skip_relocation
+    sha256 "be3d2f33e9a50cf810399c32e7c80636d63f64c17e0abaaf30fba745819166ef" => :sierra
+  end
+
+  depends_on "go" => :build
+  depends_on "glide" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GLIDE_HOME"] = HOMEBREW_CACHE/"glide_home/#{name}"
+    dir = buildpath/"src/github.com/fnproject/cli"
+    dir.install Dir["*"]
+    cd dir do
+      system "glide", "install", "-v", "--force", "--skip-test"
+      system "go", "build", "-o", "fn"
+      bin.install "fn"
+    end
+  end
+
+  def post_install
+    puts `fn --version`
+  end
+
+  test do
+    output = `#{bin}/fn --version`
+    assert_match version.to_s, output
```

Comment:
Use `assert_match version.to_s, shell_output("#{bin}/fn --version")`. Can’t we write a test that does something more substantial though?

Reply:
We have tests in our repo that run on every commit, not sure what we would do differently here?